### PR TITLE
Replace httpwg spec URLs with readable-anchor equivalents

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: http/headers/keep-alive.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Keep-Alive",
           "spec_url": [
             "https://datatracker.ietf.org/doc/html/draft-thomson-hybi-http-timeout-03#section-2",
-            "https://httpwg.org/specs/rfc7230.html#appendix-A.1.2"
+            "https://httpwg.org/specs/rfc7230.html#compatibility.with.http.1.0.persistent.connections"
```
Comment (Elchi3) [inline http/headers/keep-alive.json:9]: Sorry about the merge conflict here. I had removed the first datatracker url

File: package.json [hunk 2]
```diff
@@ -51,5 +51,9 @@
     "show-errors": "npm test 1> /dev/null",
     "test": "npm run lint && npm run mocha",
     "traverse": "node scripts/traverse"
+  },
+  "dependencies": {
```
Comment (Elchi3) [inline package.json:55]: Please undo changes to package.json

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Wonderful! Just two nits

File: http/headers/keep-alive.json (hunk 1 above)
Comment (sideshowbarker) [reply http/headers/keep-alive.json]: OK

File: package.json (hunk 2 above)
Comment (sideshowbarker) [reply package.json]: OK

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**